### ISSUE TITLE
how to use registerFilter or the range sliders together with Mixitup Multifilter

### ISSUE THREAD

**Issue by Lieneke** [patrickkunka/mixitup#575]
We are trying to implement the range sliders in our website that is using the multifilter plugin.

We implemented it according to this demo:
https://demos.kunkalabs.com/mixitup/filtering-by-range/

But when we use the registerFilter the entire multifilter stops working. We can't filter with the other filters anymore. The new range filter is the only one that works.

On the live website we are using the filter() method, but this doesn't work in combination with the other filters.
https://cultuurenschoolutrecht.nl/voor-leerlingen-po/

On the test server we have added the function registerFilter() with the date range filter, but now the other filters don't do anything anymore. 
https://csu.studionaam.com/voor-leerlingen-po/

**Comment by fxma07** [comment]
Hi! Were you able to find a solution to this? I'm having the same issue when having range sliders and multifilter.

**Comment by Lieneke** [comment]
No, we ended up creating our own javascript for this specific website. Not as fancy, but it works :)

**Comment by stratboy** [comment]
It also seems that mixitup.registerFilter is not documented..
